components/brave_rewards/resources/ui/components/modalBackupRestore/index.tsx

```diff
@@ -310,13 +311,20 @@ export default class ModalBackupRestore extends React.PureComponent<Props, State
   }
 
   getReset = () => {
+    const tags = getLocaleTags('rewardsResetTextFunds', {
+      amount: this.props.internalFunds.toString()
+    })
     return (
       <>
         <StyledTextWrapper>
           <StyledText data-test-id={'reset-text'}>
             {
               this.props.internalFunds > 0
-              ? <span dangerouslySetInnerHTML={{ __html: getLocale('rewardsResetTextFunds', { amount: this.props.internalFunds }) }} />
+              ? <span>
```

Comment:
i guess the way you would test this is to put some BAT in your wallet, then go to brave://rewards, click the settings icon, and click 'Reset'. i did not test this due to lack of BAT.